android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/UriBeaconDiscoveryService.java

```diff
@@ -282,7 +286,12 @@ public void onMdnsUrlFound(String url) {
     }
   }
 
-  private void startSearchingForUriBeacons() {
+    @Override
+    public void onSsdpUrlFound(String url) {
+        onMdnsUrlFound(url);
```

Comment:
For specificity, how about we put the code that is currently in onMdnsUrlFound:

    if (!mUrlToUrlMetadata.containsKey(url)) {
      // Fabricate the device values so that we can show these ersatz beacons
      String mockAddress = generateMockBluetoothAddress(url.hashCode());
      int mockRssi = 0;
      int mockTxPower = 0;
      mUrlToUrlMetadata.put(url, null);
      mDeviceAddressToUrl.put(mockAddress, url);
      mRegionResolver.onUpdate(mockAddress, mockRssi, mockTxPower);
      MetadataResolver.findUrlMetadata(this, UriBeaconDiscoveryService.this, url, mockTxPower, mockRssi);
    }

into another method called something like onLanUrlFound or onNonBleUrlFound. Then both on onMdnsUrlFound and onSsdpUrlFound will call that new method?